src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -365,6 +375,10 @@ export class SimpleSuggestWidget<TModel extends SimpleCompletionModel<TItem>, TI
 				}, 250);
 				const sub = token.onCancellationRequested(() => loading.dispose());
 				try {
```

Comment:
I was debugging and it looks like this part only gets called for terminal suggest items, and not the editor suggest items.
So we may be okay modifying `simpleSuggestWidget.ts`.

I think this file was also modified https://github.com/microsoft/vscode/pull/237686 too